core/scrollbar.js

```diff
@@ -306,9 +306,9 @@ Blockly.Scrollbar.prototype.resizeHorizontal_ = function(hostMetrics) {
   this.svgKnob_.setAttribute('width', Math.max(0, innerLength));
   this.xCoordinate = hostMetrics.absoluteLeft + 0.5;
   if (this.pair_ && this.workspace_.RTL) {
-    this.xCoordinate += hostMetrics.absoluteLeft +
-        Blockly.Scrollbar.scrollbarThickness;
+    this.xCoordinate += Blockly.Scrollbar.scrollbarThickness;
```

Comment:
Can you remind me why absoluteLeft is gone? Wasn't this what shifted the position of the scrollbar to the beginning (ie. right) in rtl mode?  Although, now that I look at the existing blockly playground, in rtl mode the scrollbar starts at the right in both RTL and LTR.